app/Http/Controllers/SubGroupController.php

```
@@ -0,0 +1,99 @@
+            'message' => 'Sub Group Updated Successfully',
+        ]);
+    }
+
+    /**
+     * Remove the specified resource from storage.
+     */
+    public function destroy(SubGroup $subGroup)
+    {
+        $deleted = DB::transaction(function () use ($subGroup) {
+            return $subGroup->delete();
+        });
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\DB' in method 'destroy'. (undefined)

(StaticAccess)